deploy/backplane/srep/hypershift/management-cluster/10-srep-management-cluster-project.ClusteRole.yml

```
@@ -94,3 +94,12 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - package-operator.run
+  resources:
+  - clusterpackages
+  - packages
```

```suggestion
  - '*'
```

Why limit to a subset of resources?

---

That's a fair observation, there was no specific reason other than me second-guessing myself about what would be approved. I don't think there is anything sensitive according to [docs](https://package-operator.run/docs/getting_started/api-reference/), so I've changed it to `*`